lib/manager/components/FeedFetchFrequency.js

```diff
@@ -26,28 +31,37 @@ const FeedFetchFrequency = (props: Props) => {
   const intervals = FREQUENCY_INTERVALS[fetchFrequency]
   if (!fetchInterval) fetchInterval = intervals[0]
   const messages = getComponentMessages('FeedFetchFrequency')
+  console.log(Object.keys(FREQUENCY_INTERVALS), messages);
   return (
     <div>
-      <span>{messages('fetchFeedEvery')}</span>
+      {messages('fetchFeedEvery')}
       {' '}
       <DropdownButton
```

Comment:
Using two dropdowns is probably fine for now, although a single dropdown will all options would ultimately work better.

I can imagine a user trying to adjust the frequency from say 1 day to 12 hours. Intuitively they would try to select the interval first, then adjust the frequency unit. Upon selecting 'hours' their number selection would be reset.